config/configuration.yaml

```diff
@@ -0,0 +1,13 @@
+# https://www.home-assistant.io/integrations/default_config/
```

Comment:
This seems a bit backwards to me, a python library like this can be used with any number of consumers.
Is this not working: https://developers.home-assistant.io/docs/creating_integration_manifest?_highlight=requirements#custom-requirements-during-development--testing?